src/components/form-field/form-field/form-field.ts

```diff
@@ -170,9 +170,13 @@ export class SbbFormField extends LitElement {
   }
 
   private _renderLabel(newValue: string): void {
-    let labelElement: HTMLLabelElement | undefined = Array.from(this.children)?.find(
+    if (!isBrowser()) {
+      return;
+    }
+    let labelElement = Array.from(this.children).find(
       (element) => element.tagName === 'LABEL',
-    ) as HTMLLabelElement | undefined;
+    ) as HTMLLabelElement;
```

Comment:
it can still be undefined?

Reply:
Yes, but VsCode linting marked it as superfluous (I agree with him).
But if you'd prefer I can revert it

Reply:
I've changed my mind, I think you're right in this case 😅 .
My thought process was that `undefined | null` is generally included in the type definition so there is no need to explicit it. But, in some cases like the return type of functions, you want to pass stronger information to the consumer by explicitly saying that `undefined` has a specific meaning.

A case where I would omit the ` | undefined ` is the type declaration of a variable
```ts
let labelElement: HTMLLabelElement = list.find( ... );

instead of 

let labelElement: HTMLLabelElement | undefined = list.find( ... );
```

But anyway, I feel like code style has an influence here. I don't think it's an absolute rule

Reply:
I just thought if you are casting with "as" you loose the undefined / null option, but maybe I was wrong and also not considering that "let" was used here

Reply:
Anyway, I've reverted the change 😅